src/phoenix/trace/dsl/query.py

```diff
@@ -404,3 +629,73 @@ def from_dict(
                 else {}
             ),
         )
+
+
+def _get_spans_dataframe(
+    session: Session,
+    span_filter: SpanFilter,
+    project_name: str,
+    start_time: Optional[datetime] = None,
+    stop_time: Optional[datetime] = None,
+    root_spans_only: Optional[bool] = None,
+) -> pd.DataFrame:
+    # legacy labels for backward-compatibility
+    span_id_label = "context.span_id"
+    trace_id_label = "context.trace_id"
```

Comment:
Not part of this PR, but do we need to keep these around or are we ready to remove?

Reply:
I don't think we can remove these without breaking backward compatibility